### ISSUE TITLE
Locations

### ISSUE THREAD

**Issue by arjunv27** [roomOnDemand/healthCareToHotels#1]
Where can healthcare workers stay during this event:
  * Hotels?
  * etc....

**Comment by shubhe25p** [comment]
renting separate spaces for healthcare workers

**Comment by shubhe25p** [comment]
> renting separate spaces for healthcare workers

> Where can healthcare workers stay during this event:
> 
> * Hotels?
> * etc....

Hotels could be a less safer option

**Comment by arjunv27** [comment]
We need to discover what spaces are available

**Comment by klunawat-realestate** [comment]
See if spaces here make sense: https://loftyspot.com/

**Comment by klunawat-realestate** [comment]
Separately - something to run by your mom - would families be okay with physicians self-isolating themselves? (ie wouldn't they want them to come home?)

**Comment by klunawat-realestate** [comment]
If case above is correct, hotel rooms may not be bad bet. They are largely vacant now (no one' staying) and can allocate rooms with sufficient space

**Comment by arjunv27** [comment]
> If case above is correct, hotel rooms may not be bad bet. They are largely vacant now (no one' staying) and can allocate rooms with sufficient space

This was my primary thought process

**Comment by arjunv27** [comment]
> See if spaces here make sense: https://loftyspot.com/

We also need spots for quarantine, i.e. convention centers, stadiums, school gymnasiums, etc

These are two separate issues imo

**Comment by klunawat-realestate** [comment]
Agreed. For physician stay, these short term rental companies are desperate to find people to stay at their units. Sure they can accommodate physicians:

https://zeusliving.com/
https://www.sonder.com/
https://www.staydomio.com/
https://www.lyric.com/

Thoughts?

**Comment by marycourtland** [comment]
I can see us having a common model/back end for locations, for both issues, and then have each location declare whether it's for hospital overflow, quarantine, or worker isolation.

**Comment by arjunv27** [comment]
> short term rental companies

Certainly an opportunity, cost need to be subsidized.

>model/back end for locations
Thoughts on integrating w/ https://developers.google.com/maps/documentation

**Comment by klunawat-realestate** [comment]
Gonna write to these short term rental companies and see what's the best they can do. 

Additionally, would hospitals be willing to subsidize part of the cost?

**Comment by arjunv27** [comment]
> subsidize

Hospital resources are stretched thin, best bet would be legislation, money is being spent for things like this -- just need to find the source.

**Comment by klunawat-realestate** [comment]
good point. private sector should be chipping in. Ex: Wall St.

**Comment by klunawat-realestate** [comment]
Or crowdsource? "Help fund a physician stay for 30 days close to her hospital"

**Comment by arjunv27** [comment]
> crowdsource

That would be a good idea, just need an MVP for now

**Comment by marycourtland** [comment]
> Thoughts on integrating w/ https://developers.google.com/maps/documentation

@arjunv27 Google maps is great - we should use it for sure. (At least for the first year of usage; then it gets quite expensive. Outside our time frame though!)

For back end, I meant, where does our data live & how it is accessed. I think we can allow a hotel to sign up before it's decided whether it's needed for quarantine or hospital overflow. Then match it up with whatever is in demand.

Alternatively this could be a completely different app from the "hospital overflow" matching app.

**Comment by arjunv27** [comment]
>For back end, I meant, where does our data live & how it is accessed.

Let's continue this discussion here: Data Models

**Comment by klunawat-realestate** [comment]
For later: one question that may come up from STR operators is to ensure that their spaces / discounted rates are only being used by medical professionals. Build some ID/authentication for that?

**Comment by arjunv27** [comment]
>ID/authentication 
Create a new issue
